app/src/lib/api.ts

```diff
@@ -374,6 +374,12 @@ export class API {
       return await parsedResponse<IAPIRepository>(response)
     } catch (e) {
       if (e instanceof APIError) {
+        const orgsList: ReadonlyArray<IAPIUser> | null = await this.fetchOrgs()
```

Comment:
Do we need to re-fetch the organizations here? I think if we tried to `POST` to `orgs/${org.login}/repos` and got a 404 here it's reasonable to think this is related to the organization.

Reply:
Wonderful, I was hoping there was a way to avoid fetching again. To clarify: if the org used is `orgs/${org.login}/repos` and received an APIError from this POST request then we have enough information to throw this error, and therefore no additional checks are needed?

Reply:
@Daniel-McCarthy yeah, I wouldn't worry about being cleverer here than that